### ISSUE TITLE
invalid index in Codeception/Step.php

### ISSUE THREAD

**Issue by tom-schumann** [Codeception/Codeception#5231]
#### What are you trying to achieve?
In Codeception/Step.php::addMetaStep there is the line codecept_debug(get_class($step['object']));
It seems that in some cases, $step contains no 'object' key (there is a check for its existence further below before it calls $this->metaStep->setPrefix) - this causes an exception and it seems to happen when calling $I->grabCookie or $I->resetCookie (but not for other FunctionalTester actions).

#### What do you get instead?

Calling $I->grabCookie results in [yii\base\ErrorException] Undefined index: object
More trace:
  [yii\base\ErrorException] Undefined index: object#0 /www/sites/dev.lan/vendor/codeception/codeception/src/Codeception/Step.php(308): yii\base\ErrorHandler->handleError(8, 'Undefined index...', '/www/sites/dev.l...', 308, Array)
#1 /www/sites/dev.lan/vendor/codeception/codeception/src/Codeception/Step.php(59): Codeception\Step->addMetaStep(Array, Array)
#2 /www/sites/dev.lan/vendor/codeception/codeception/src/Codeception/Scenario.php(64): Codeception\Step->saveTrace()
#3 /www/sites/dev.lan/backend/tests/_support/_generated/FunctionalTesterActions.php(1136): Codeception\Scenario->runStep(Object(Codeception\Step\Action))

### Details

* Codeception version: 2.5.0
* PHP Version: 7.2.11
* Operating System: CentOS 7.5.1804 (in a Docker container)
* Installation type: Composer
* List of installed packages: Yii 2.0.13.1, c3 2.4.1 etc
* Suite configuration:

```yml
class_name: FunctionalTester
modules:
    enabled:
        - Db
        - Asserts
        - WebDriver:
            url: 'https://dev.lan/index-test.php'
            host: selenium
            port: 4444
            browser: chrome
            window_size: 1920x1080
            log_js_errors: true
            capabilities:
                acceptSslCerts: true
        - Yii2:
            part: [init, orm, fixtures, email]
            cleanup: false
        - REST:
            depends: [Yii2]
            url: 'https://dev.lan/index-test.php'

```

**Comment by ruudy-es** [comment]
Facing same issue with Laravel 5.5

**Comment by tom-schumann** [comment]
Seems like it's fixed in Codeception 2.5.1
